### PR TITLE
change footer text format to minimal

### LINKED ISSUE #4271: Update footer text area to use minimal html text format
We will want to plan a communication around this as part of this issue as well.

Related to #4213

### PR DESCRIPTION
**Pull request by makurj**
Relates to #4213 
Resolves https://github.com/uiowa/uiowa/issues/4271

### CODE REVIEW

**Comment by dnkemper** [comment]
HOW DID THIS HAPPEN FROM MY PR?

**Comment by joewhitsitt** [comment]
```
FILE: ...uild/uiowa/uiowa/docroot/profiles/custom/sitenow/sitenow.install
----------------------------------------------------------------------
FOUND 9 ERRORS AFFECTING 9 LINES
----------------------------------------------------------------------
 3763 | ERROR | [x] Expected 1 space(s) before asterisk; 0 found
 3764 | ERROR | [x] Expected 1 space(s) before asterisk; 0 found
 3766 | ERROR | [x] Line indented incorrectly; expected 2 spaces,
      |       |     found 1
 3767 | ERROR | [x] Line indented incorrectly; expected 2 spaces,
      |       |     found 1
 3768 | ERROR | [x] Line indented incorrectly; expected 4 spaces,
      |       |     found 3
 3770 | ERROR | [x] Line indented incorrectly; expected 2 spaces,
      |       |     found 1
 3771 | ERROR | [x] Line indented incorrectly; expected 4 spaces,
      |       |     found 3
 3772 | ERROR | [x] Line indented incorrectly; expected 4 spaces,
      |       |     found 3
 3773 | ERROR | [x] Line indented incorrectly; expected 2 spaces,
      |       |     found 1
----------------------------------------------------------------------
PHPCBF CAN FIX THE 9 MARKED SNIFF VIOLATIONS AUTOMATICALLY
```

File: docroot/profiles/custom/sitenow/sitenow.profile [hunk 1]
```diff
@@ -485,15 +485,26 @@ function sitenow_form_alter(&$form, FormStateInterface $form_state, $form_id) {
...
       break;
 
-    // Remove access to headline field in footer contact block.
     case 'block_content_uiowa_text_area_edit_form':
       $block = $form_state->getFormObject()->getEntity();
       $uuid = $block->uuid();
-      // For Footer Contact Information, remove field.
+      // For Footer Contact Information, limit non-admins to minimal.
       if ($uuid == '0c0c1f36-3804-48b0-b384-6284eed8c67e') {
-        $form['field_uiowa_headline']['#access'] = FALSE;
```
Comment (joewhitsitt) [inline docroot/profiles/custom/sitenow/sitenow.profile:494]: Looks like your merge removed our work to remove the headline field from this block

**Comment by GaryRidgway** [comment]
@joewhitsitt the headline removal has been re-added

**Review by joewhitsitt — APPROVED** [review]
Thanks for dusting this off @GaryRidgway. Nice working on this with you @makurj

**Pull request merged**